src/output-json-flow.c

```diff
@@ -272,6 +272,8 @@ static void EveFlowLogJSON(OutputJsonThreadCtx *aft, JsonBuilder *jb, Flow *f)
     jb_set_bool(jb, "alerted", FlowHasAlerts(f));
     if (f->flags & FLOW_WRONG_THREAD)
         JB_SET_TRUE(jb, "wrong_thread");
+    if (FlowHasGaps(f))
```

Comment:
since gaps are per direction and for TCP only, we should probably log them in the `tcp` object below